packages/java/endpoint/src/main/java/com/vaadin/hilla/Nonnull.java

```diff
@@ -28,9 +28,13 @@
  *
  * This annotation exists only for convenience because the traditional
  * `jakarta.annotation.Nonnull` annotation is not applicable to type parameters.
+ *
+ * @deprecated use the standardized {@link org.jspecify.annotations.NonNull}
+ *             instead
  */
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ ElementType.TYPE_USE })
+@Deprecated(forRemoval = true)
```

Comment:
Why is `Nonnull` deprecated and not `Nullable`? The same should apply.

Reply:
Added deprecation